WordPress/Classes/Extensions/String+Helpers.swift

```diff
@@ -28,6 +28,12 @@ extension String {
         return copy
     }
 
+    func stripHtmlTags() -> String {
```

Comment:
No problem, but I think [Swift 3 renamed the obj-C method ](https://github.com/apple/swift-evolution/blob/master/proposals/0005-objective-c-name-translation.md) to `strippingHTML`